### ISSUE TITLE
segmentation fault (core dumped)

### ISSUE THREAD

**Issue by alexrider1105** [xingyizhou/CenterTrack#120]
Hi, upon running the following:
python demo.py tracking,ddd --load_model ../models/nuScenes_3Dtracking.pth --dataset nuscenes --pre_hm --track_thresh 0.1 --demo ../videos/nuscenes_mini.mp4 --test_focal_length 633
I receive the following output in the command line:
/home/shihkuan/anaconda3/envs/CenterTrack/lib/python3.6/site-packages/sklearn/utils/linear_assignment_.py:21: DeprecationWarning: The linear_assignment_ module is deprecated in 0.21 and will be removed from 0.23. Use scipy.optimize.linear_sum_assignment instead.
  DeprecationWarning)
Running tracking
Using tracking threshold for out threshold! 0.1
Fix size testing.
training chunk_sizes: [32]
input h w: 448 800
heads {'hm': 10, 'reg': 2, 'wh': 2, 'tracking': 2, 'dep': 1, 'rot': 8, 'dim': 3, 'amodel_offset': 2}
weights {'hm': 1, 'reg': 1, 'wh': 0.1, 'tracking': 1, 'dep': 1, 'rot': 1, 'dim': 1, 'amodel_offset': 1}
head conv {'hm': [256], 'reg': [256], 'wh': [256], 'tracking': [256], 'dep': [256], 'rot': [256], 'dim': [256], 'amodel_offset': [256]}
Creating model...
Using node type: (<class 'model.networks.dla.DeformConv'>, <class 'model.networks.dla.DeformConv'>)
Warning: No ImageNet pretrain!!
loaded ../models/nuScenes_3Dtracking.pth, epoch 70
Initialize tracking!
Segmentation fault (core dumped)
Has anyone had a similar issue? I am hoping that someone can help me resolve this issue.

**Comment by xingyizhou** [comment]
Can you provide your pytorch/ cuda version and how you install DCNv2? I guess this is a DCN installation issue.

**Comment by kuanhc96** [comment]
Hi @xingyizhou , I am using CUDA 10.1, pytorch 1.0.0, and torchvision 0.2.1.
As for DCNv2, I navigated to $CenterTrack_ROOT/src/lib/model/networks/ and called git clone https://github.com/CharlesShang/DCNv2/, then I called ./make.sh.
Thanks for your time! Please let me know your thoughts!

**Comment by Pirazh** [comment]
I used Pytorch 1.0 and CUDA 10.0 to get it to work, also note that when installing dependancies in the `requirements.txt`, "nuscenes-devkit" dependency removes torch and installs the latest version of it which you do not want. In my case I did want nuScenes dataset so I removed it.
